react-spinkit/index.d.ts

```diff
@@ -1,18 +1,21 @@
 // Type definitions for react-spinkit 1.1.4
 // Project: https://github.com/KyleAMathews/react-spinkit
-// Definitions by: Qubo <https://github.com/tkqubo>
+// Definitions by: Qubo <https://github.com/tkqubo>, Mleko <https://github.com/mleko>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.1
 
 /// <reference types="react" />
 
+declare namespace ReactSpinkit {
+	interface Props {
+		spinnerName?: string;
+	}
 
-import { Component } from 'react';
-
-interface Props {
-    spinnerName?: string;
+	interface Spinner extends React.ComponentClass<Props> { }
+}
+declare module "react-spinkit" {
+	var spinner: ReactSpinkit.Spinner;
+	export = spinner;
 }
```

Comment:
Proper export style should be an top-level `export = spinner`. Rename your namespace to `spinner` so that you automatically export the interfaces as well.